backfiller/backfiller/main.py

```diff
@@ -383,9 +400,11 @@ def run(self):
 @argh.arg('--node-file', help="Name of file listing nodes to backfill from. One node per line in the form NAME URI with whitespace only lines or lines starting with '#' ignored. If None (default) do not get nodes from a file.")
 @argh.arg('--node-database', help='Postgres conection string for database to fetch a list of nodes from. Either a space-separated list of key=value pairs, or a URI like: postgresql://USER:PASSWORD@HOST/DBNAME?KEY=VALUE . If None (default) do not get nodes from database.')
 @argh.arg('--localhost', help='Name of local machine. Used to prevent backfilling from itself. By default the result of socket.gethostname()')
+@argh.arg('--download-concurrency', help='Max number of concurrent segment downloads from a single node. Increasing this number may increase throughput but too high a value can overload the server or cause timeouts.')
```

Comment:
Should mention the default value in the doc string.

Reply:
When the usage is printed, the default value is included automatically:
```
  --download-concurrency DOWNLOAD_CONCURRENCY
                        Max number of concurrent segment downloads from a
                        single node. Increasing this number may increase
                        throughput but too high a value can overload the
                        server or cause timeouts. (default: 5)
```